App/CrisisCleanup/Sources/Feature/Cases/UI/CasesView.swift

```diff
@@ -1,12 +1,46 @@
 import SwiftUI
+import SVGView
 
 struct CasesView: View {
     @Environment(\.translator) var t: KeyAssetTranslator
 
     @ObservedObject var viewModel: CasesViewModel
+    let authenticateViewBuilder: AuthenticateViewBuilder
     let incidentSelectViewBuilder: IncidentSelectViewBuilder
+    let incidentsTypeIconsAssetsFolder = "incident_type_icons/"
 
-    var body: some View {
-        Text(t("casesVue.cases"))
+    @State var showIncidentSelect = false
+
+    var body: some View{
+        VStack {
+            HStack {
+                Button {
+                    showIncidentSelect.toggle()
+                } label: {
+                    HStack{
+                        Image(incidentsTypeIconsAssetsFolder+viewModel.incidentsData.selected.disasterLiteral, bundle: .module).foregroundColor(Color.blue)
```

Comment:
Heads up here and for the MenuView as well. `viewModel.incidentsData.selected.disasterLiteral` is a blank string until an incident loads. If you observe the app longs you'll notice a message about how an image cannot be found when this string is blank.